xlsxwriter.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@PREFIX@
```

Comment:
All pkg-config files have this naming convention where the file is named after the library name and the `.in` suffix indicates that it's a template. CMake uses that to generate the `.pc` file. I thought about adding `autoconf` and `automake` configurations to generate `.pc` files from the template but it was too much given the simplicity of the project so I went with good old `sed`.

 I think naming it `pkg-config.txt` is confusing because pkg-config files have a `.pc` extension, see `ls /usr/local/lib/pkgconfig` or `ls /usr/lib/pkgconfig` if on Linux. Some packages like [libyaml](https://github.com/yaml/libyaml/blob/master/yaml-0.1.pc.in) place the file in the root directory and some like [harfbuzz](https://github.com/harfbuzz/harfbuzz/blob/master/src/harfbuzz.pc.in) place it in the `src` directory. I think placing it in the root directory is the right place cause the file is not related to the source code.

Reply:
> I think naming it pkg-config.txt is confusing because pkg-config files have a .pc

Just to be clear. I'm not suggesting renaming the output/target file, that can/should be xlsxwriter.pc. My issue is with the name of the template file. For me `*.in` indicated autoconf usage, which isn't the case here. The example you show both use autoconf so that is acceptable in their case.

So in summary I'd prefer the template file to be `dev/release/pkg-config.txt`.